### DISCUSSION TITLE
Policy or framework to manage participation in tech conferences

### DISCUSSION THREAD

**Discussion by jsmanrique** [todogroup/ospology#35] [OSPO Discussions]
As an OSPO, we think we need a *framework or policy for our developers to participate in tech conferences* that IMHO should cover at least:
* How to request participation?
* Which levels of participation are possible?
* Code of conduct?
* Which resources are available for participants?
* Content to be shared review procedures? 
* Dress code (we are a fashion company 😄  )?
* How to share experience and knowledge acquired

Have you seen something similar in your OSPOs? Any reference?

**Comment by anajsana** [comment]
That can be an interesting guide we can build with other TODO folks! I've opened an issue https://github.com/todogroup/todogroup.org/issues/263

**Comment by mwichmann** [comment]
I'll render a (probably unpopular) opinion: I'd strive for framing it as **Guidance** for participation, rather than **Policy/Process**.  The former feels more like encouraging me to make a compelling proposal to participate, while the latter sounds more like discouraging me because there are all these things to make it hard.  Obviously, if you're participating on the company's dime (dime: we wish!) there are expectations.  The document should start with some description of *why* the company encourages such participation - obviously the IC gets something out of it, and maybe that's part of the company motivation, but usually there's more.
